tests/tensor/test_math_opt.py

```diff
@@ -2742,26 +2742,19 @@ def setup_method(self):
         self.mode = (
             get_default_mode()
             .including("canonicalize", "fast_run")
-            .excluding("gpu", "fusion")
+            .excluding("gpu", "fusion", "inplace")
         )
-        self.mode._optimizer.position_cutoff = 1.50001
 
     def test_local_one_plus_erf(self):
         val = np.asarray([-30, -3, -2, -1, 0, 1, 2, 3, 30], dtype=config.floatX)
         x = vector()
 
         f = function([x], 1 + erf(x), mode=self.mode)
-        assert [n.op for n in f.maker.fgraph.toposort()] == [
-            mul,
-            erfc,
-        ], f.maker.fgraph.toposort()
+        assert [n.op for n in f.maker.fgraph.toposort()] == [neg, erfc]
```

Comment:
The assert prints were not really helpful, since they would all just be `Elemwise<...>`

Reply:
Understandable, but these changes belong in a separate commit.